### PR TITLE
Add GitHubTeacher Orlando file to project

### LINKED ISSUE #2: GitHubTeacher tracks steps using GitHub Flow
### GitHubTeacher adds Orlando, FL

- [x] create a branch
- [x] create some commits
- [x] open a pull request
- [x] collaborate + additional commits
- [x] merge

### PR DESCRIPTION
**Pull request by githubteacher**
This PR closes #2.

This PR is for the purposes of adding the Orlando, FL file to the hometown files. 

Hey @hectorsector -- could you put some 👀 on this PR, please? Give me a 👍  if you think it's good to 🚢 . Thanks 🙇!

### CODE REVIEW

File: githubteacher-orlando.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Orlando, FL
+
+### Restaurants
+
+- The Pharmacy
+- 4Rivers
+
+### Tourist Attractions
```
Comment (hectorsector) [inline githubteacher-orlando.md:8]: @githubteacher Orlando is big for Theme Parks -- isn't it? Shouldn't you include that?

**Comment by githubteacher** [comment]
@hectorsector -- could you review this before I 🚢?

File: githubteacher-orlando.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Orlando, FL
+
+### Restaurants
+
+- The Pharmacy
+- 4Rivers
```
Comment (hectorsector) [inline githubteacher-orlando.md:6]: 👍 I ❤️ 4Rivers!

File: githubteacher-orlando.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+### Tourist Attractions
+
+- Disney World
+- Universal Studios Orlando
+- Legoland
+
+### Local Attractions
+
+- Gatorland
+- Payne's Praire in Gainesville
```
Comment (hectorsector) [inline githubteacher-orlando.md:17]: @githubteacher Gainesville is pretty far away -- maybe you shouldn't include it? But it's not a blocker.

**Review by hectorsector — APPROVED** [review]
@githubteacher great Orlando file -- I say :shipit:

**Pull request merged**